client/src/components/Dialogs/PostReply.js

```diff
@@ -30,6 +30,16 @@ function ReplyText(props) {
 export default function PostReply(props) {
   const [state, setState] = useState({ isEditing: false, text: props.reply.text })
 
+  const removeReplyFunc = arg => {
+  //   const newReplies = [...oldReply];
+  //   const newReplyAfterRemovals = newReplies.filter(reply => reply.id !== arg);
+  //   setOldReply(newReplyAfterRemovals);

```

Comment:
You can use this to create a `deleteReply` handler in the parent of `PostReply`. Do that inside the `replies.map` function so you can bind each reply and its id to the handler.